test/bdd/steps/api/resolve.mjs

```diff
@@ -1,19 +1,25 @@
 import { When, Given } from '@cucumber/cucumber';
 import { expect, assert } from 'chai';
 import { setTimeout } from 'timers/promises';
+import {createRequire} from "module";
+import HttpApiHelper from "../../../utilities/http-api-helper.mjs";
+
+
+const require = createRequire(import.meta.url);
+const requests = require('./datasets/requests.json');
```

Comment:
We should use import from not require, can you please update this

Reply:
I can use **import config from "./datasets/requests.json." assert {type:"json"}**,  but that will throw a warning:

_(node:2642608) ExperimentalWarning: Importing JSON modules is an experimental feature. This feature could change at any time_ 

Importing like this: **import file from "./some/path/file.json"**; is still not possible as [Importing JSON using ES modules ](https://github.com/tc39/proposal-json-modules) is still in stage 3 of ECMAScript proposals.

Should i leave it like this or use the experimental feature?

Reply:
I would say to remove require completely, you can also use fileService to read this file from path. So try to do that instead of using experimental feature